.github/workflows/main.yml

```diff
@@ -87,4 +87,12 @@ jobs:
         if [ -e "server/firebase-secret.json" ] || [ -e ".env" ]; then
           echo "Error: Found .env or firebase-secret.json in the pull request. Please remove them before merging."
           exit 1
-        fi
\ No newline at end of file
+        fi
+
+    - name: Check Prettier Formatting
```

Comment:
can you make this its own separate job? Right now its running underneath the file existence job.

Reply:
can you pls check now